pkg/scanner/scanner.go

```diff
@@ -206,13 +206,6 @@ func (s *Scanner) Scan(ctx context.Context, cloudState *state.State) (results sc
 	return append(results, regoResults...), nil
 }
 
-func (s *Scanner) getRules() []types.RegisteredRule {
-	if len(s.frameworks) > 0 { // Only for maintaining backwards compat
-		return rules.GetRegistered(s.frameworks...)
-	}
-	return rules.GetSpecRules(s.spec)
-}
-
```

Comment:
I think we can revert this and import RegisteredRule from trivy-iac.

Reply:
did so https://github.com/aquasecurity/trivy-aws/pull/35/commits/b89962ae3b5231d3c77b7a71e3bcd18d6f24c3a0